DatadogSessionReplay/README.md

```diff
@@ -1,3 +1,3 @@
 # Session Replay
 
-🚧 The Session Replay feature is under construction.
+🚧 The Session Replay feature is currently in beta - contact Datadog to request a try.
```

Comment:
Link to contact form or email would be great.

Reply:
I'd add it once the process is established (public beta). For now (closed beta) there's IMO not much sense.